src/app/tabs/registrar/actions.js

```diff
@@ -103,3 +104,12 @@ export const toggleSetupAddr = setupAddr => ({
 export const closeErrorMessage = () => ({
   type: CLOSE_REGISTRATION_ERROR,
 });
+
+export const requestHasEnoughRif = () => ({
+  type: REQUEST_HAS_ENOUGH_RIF,
+});
+
+export const errorHasEnoughRIF = message => ({
```

Comment:
should 'errorHasEnoughRIF' be renamed to 'errorNotEnoughRIF' to be more clear?

Reply:
done!